### PR TITLE
remove cassandraImpl and added new changes

### PR DESCRIPTION
**Pull request by Manas-tarento**

### CODE REVIEW

File: src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java [hunk 1]
```diff
@@ -641,4 +633,24 @@ public Response getUserWFApplicationFields(String rootOrg, String org, String wi
...
+			if (serviceName.equalsIgnoreCase(Constants.PROFILE_SERVICE_NAME)) {
+				StringBuilder uri = new StringBuilder();
+				uri.append(configuration.getLmsServiceHost() + configuration.getWfSystemSettingsPath());
+				wfConfig = (Map<String, Object>) requestServiceImpl.fetchResultUsingGet(uri);
+			}
+			Map<String, Object> result = (Map<String, Object>) wfConfig.get(Constants.RESULT);
+			Map<String, Object> response = (Map<String, Object>) result.get(Constants.RESPONSE);
+			String value = (String) response.get(Constants.VALUE);
+			Map<String,Object> wfStates = mapper.readValue(value,Map.class);
+			WorkFlowModel workFlowModel = mapper.convertValue(wfStates, WorkFlowModel.class);
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java]: Use TypeReference class. Refer cb-ext service for example.

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -34,6 +28,7 @@ lms.user.update=private/user/v1/update
 lms.user.read=private/user/v1/read/userIdValue
 lms.user.migrate=private/user/v1/migrate 
 lms.assign.role=private/user/v1/assign/role
+lms.system.settings.path=v1/system/settings/get/wfProfileServiceConfig
```
Comment (karthik-tarento) [inline src/main/resources/application.properties]: change the variable name -- to mention that this is wfProfileService config path

File: src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java [hunk 3]
```diff
@@ -641,4 +633,24 @@ public Response getUserWFApplicationFields(String rootOrg, String org, String wi
...
+		try {
+			Map<String, Object> wfConfig = new HashMap<>();
+			if (serviceName.equalsIgnoreCase(Constants.PROFILE_SERVICE_NAME)) {
+				StringBuilder uri = new StringBuilder();
+				uri.append(configuration.getLmsServiceHost() + configuration.getWfSystemSettingsPath());
+				wfConfig = (Map<String, Object>) requestServiceImpl.fetchResultUsingGet(uri);
+			}
+			Map<String, Object> result = (Map<String, Object>) wfConfig.get(Constants.RESULT);
+			Map<String, Object> response = (Map<String, Object>) result.get(Constants.RESPONSE);
+			String value = (String) response.get(Constants.VALUE);
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java]: variable value is not required

File: pom.xml [hunk 4]
```diff
@@ -73,6 +65,11 @@
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <version>r05</version>
+        </dependency>
```
Comment (karthik-tarento) [inline pom.xml]: this package is not used anywhere ??

File: src/main/java/org/sunbird/workflow/controller/TaxonomyWorkFlowController.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.sunbird.workflow.models.Response;
+import org.sunbird.workflow.models.WfRequest;
+import org.sunbird.workflow.service.Workflowservice;
+
+@RestController
+@RequestMapping("/v1/workflow/taxonomy")
+public class TaxonomyWorkFlowController {
```
Comment (karthik-tarento) [inline src/main/java/org/sunbird/workflow/controller/TaxonomyWorkFlowController.java]: No need to add this class in this PR

**Review by karthik-tarento — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 4 above)
Comment (Manas-tarento) [reply pom.xml]: This thing used in WFLogger class . I removed the Cassandra dependencies that's why  "com.google.common.base.Throwables;"  this package giving error.

File: src/main/java/org/sunbird/workflow/controller/TaxonomyWorkFlowController.java (hunk 5 above)
Comment (Manas-tarento) [reply src/main/java/org/sunbird/workflow/controller/TaxonomyWorkFlowController.java]: ok
will remove the class

**Review by karthik-tarento — APPROVED** [review]

**Pull request merged**